src/theme/theme.ts

```diff
@@ -399,316 +399,170 @@ const themeCustomized: Theme = createTheme({
       'sans-serif',
     ].join(','),
     lifiHeaderDisplay: {
-      fontFamily: [
```

Comment:
Is it to avoid loading too many fonts in the app?

Reply:
No, it´s more: That fontFamily is already being set as default and here I repeatedly defined it again for no reason.